docs/checkouts/lifecycle.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: Checkout lifecycle
+sidebar_label: Lifecycle
+description: Understand a Checkout lifecycle
+keywords:
+  - checkouts
+  - features
+  - lifecycle
+---
+
+import Figure from "@site/src/components/Figures";
+import CheckoutStatuses from "@site/static/img/checkout_statuses.png";
+
+# Lifecycle
+
+## Overview
+
+A Checkout is not a Payment; it has its own data model, lifecycle, and statuses.
+
+### Diagram
+
+- When a Checkout is created it status is set to `created`.
+- If a payment attempt fails, it transitions to the status `attempted`
+  and remains there until it either expires or a payment is successful.
+- If a payment succeeds, it transitions to the status `completed`.
+- If not completed within 1 week, it automatically transitions to the status `expired`.
+
+<Figure
+  caption="The Checkout states and transitions."
+  alt="Checkout Statuses"
+  src={CheckoutStatuses}
+/>
+
+### Datamodel
+
+- A Checkout has its own attributes set at creation and can be updated if not completed or expired.
```

Comment:
Hm, I don't think we are consistent all through the docs on when to use capitalized or lower case for our models/API endpoints.
I have been using lower case (in this case it would be 'A checkout....') unless I link to a specific section.